Contextify 0.1.10 fails to install on node 0.11.14

```
$ node --version
v0.11.14
```

```
$ npm install contextify@v0.1.9
-
> contextify@0.1.9 install /Users/myself/stuff/node_modules/contextify
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
  CXX(target) Release/obj.target/contextify/src/contextify.o
  SOLINK_MODULE(target) Release/contextify.node
  SOLINK_MODULE(target) Release/contextify.node: Finished
contextify@0.1.9 node_modules/contextify
├── bindings@1.2.1
└── nan@1.3.0
```

```
$ npm install contextify@v0.1.10

> contextify@0.1.10 install /Users/myself/stuff/node_modules/contextify
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
  CXX(target) Release/obj.target/contextify/src/contextify.o
../src/contextify.cc:138:44: error: member reference type 'Persistent<v8::Function>' is not a pointer; maybe you meant to use '.'?
        Local<Object> wrapper = constructor->NewInstance();
                                ~~~~~~~~~~~^~
                                           .
../src/contextify.cc:138:46: error: no member named 'NewInstance' in 'v8::Persistent<v8::Function, v8::NonCopyablePersistentTraits<v8::Function> >'
        Local<Object> wrapper = constructor->NewInstance();
                                ~~~~~~~~~~~  ^
2 errors generated.
make: *** [Release/obj.target/contextify/src/contextify.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1059:12)
gyp ERR! System Darwin 14.0.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/myself/stuff/node_modules/contextify
gyp ERR! node -v v0.11.14
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok 

npm ERR! Darwin 14.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "contextify@v0.1.10"
npm ERR! node v0.11.14
npm ERR! npm  v2.0.0
npm ERR! code ELIFECYCLE
npm ERR! contextify@0.1.10 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the contextify@0.1.10 install script.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.
```

--- a/changelog
+++ b/changelog
@@ -1,3 +1,5 @@
+0.1.11
+    * Fix build error on node 0.11 (Sergei Ianovich - @yanovich)
 0.1.10
     * Fix segfault in GlobalPropertyGetter() (Sergei Ianovich - @yanovich)
 0.1.9

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "contextify",
-    "version": "0.1.10",
+    "version": "0.1.11",
     "description": "Turn an object into a persistent execution context.",
     "author": "Brian McDaniel <brianmcd05@gmail.com>",
     "contributors": [ 

--- a/src/contextify.cc
+++ b/src/contextify.cc
@@ -135,7 +135,7 @@ public:
 
     static Local<Context> createV8Context(Handle<Object> jsContextify) {
         NanEscapableScope();
-        Local<Object> wrapper = constructor->NewInstance();
+        Local<Object> wrapper = NanNew(constructor)->NewInstance();
 
         ContextWrap *contextWrapper = new ContextWrap();
         contextWrapper->Wrap(wrapper);

--- a/wscript
+++ b/wscript
@@ -2,7 +2,7 @@ import Options
 import os
 import sys
 
-VERSION = '0.1.10'
+VERSION = '0.1.11'
 
 def set_options(opt):
   opt.tool_options("compiler_cxx")

